### ISSUE TITLE
Exagear wine emulator

### ISSUE THREAD

**Issue by GpS1979** [Pyrdacor/Ambermoon.net#336]
Hello. I'd like try the game on android using Exagear wine 7.2 emulator. I downloaded win32 version but crash at start. It will be great having the opportunity to play the game with tablet or mobile so i'm asking if there is a correct config for this problem. thanks,

**Comment by a1exh** [comment]
@Pyrdacor will know for sure but I don't think you can do this. 

Ambermoon.NET is not a stand-alone windows application. It needs the .NET runtime.

You'd somehow have to install .NET in Wine first.

**Comment by a1exh** [comment]
I think @Pyrdacor was looking at porting the Linux version to Android

**Comment by GpS1979** [comment]
thanks. last  .net   is already installer , seems the problem is another! I don't know if there is a log files i can share.

**Comment by Pyrdacor** [comment]
Hi. It you run it from a terminal you should see some error printout. You might need glfw and OpenAL installed as I don't know if the dependency resolution works as expected with wine. And ensure that your user has write access to the folders etc.

**Comment by Pyrdacor** [comment]
> I think @Pyrdacor was looking at porting the Linux version to Android

Yeah creating an Android version is part of my ever growing todo list.

**Comment by Pyrdacor** [comment]
This is kinda old but Ambermoon is now available for Android via Google Playstore or from itch.io (https://pyrdacor.itch.io/ambermoon).
